rosie.py

```diff
@@ -1,4 +1,20 @@
+"""Rosie.
+
+Usage:
+    python rosie.py run (chamber_of_deputies | federal_senate)
+    python rosie.py run (chamber_of_deputies | federal_senate) [--years 2017,2016]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>] [--years 2017,2016]
+    python rosie.py test [chamber_of_deputies | federal_senate]
+
+Options:
+    --years   runs rosie wuth years filter
```

Comment:
Typo: `wuth` -> `with`

Rosie is a proper noun and should be capitalized.

In _years filter_ the word _years_ is an adjective (I guess) and should not be pluralized.

Also _runs rosie wuth years filter_ doesn't tell much about what _years filter_ does. What about: _runs Rosie fetching data and finding suspicions for a specific set of years_?

Reply:
accepted 👍